### PR TITLE
Issue 3318: Make sure that we fetch an author

### PR DESCRIPTION
**Pull request by annando**
This possibly could fix https://github.com/friendica/friendica/issues/3318 - or not. Since I can't reproduce that issue I can only guess and create some additional checks. Hopefully this helps.

### CODE REVIEW

File: include/ostatus.php [hunk 1]
```diff
@@ -2288,6 +2313,10 @@ public static function salmon($item,$owner) {
 		$doc = new DOMDocument('1.0', 'utf-8');
 		$doc->formatOutput = true;
 
+		if (Config::get('system', 'ostatus_debug')) {
+			$item['body'] .= '🐟';
```
Comment (Hypolite) [inline include/ostatus.php]: 😂

File: include/ostatus.php [hunk 2]
```diff
@@ -2268,6 +2290,9 @@ public static function feed(App $a, $owner_nick, $last_update) {
 		$root = self::add_header($doc, $owner);
 
 		foreach ($items AS $item) {
+			if (Config::get('system', 'ostatus_debug')) {
+				$item['body'] .= '🍼';
```
Comment (Hypolite) [inline include/ostatus.php]: 😂

**Comment by annando** [comment]
What do you mean with "rerun"? Export or import? With some effort (some external code) you can generate the XML for the export. The import would be possible with some effort (additional programming) as well.

**Comment by Hypolite** [comment]
I'm thinking about Import, as we are saving the actual XML message, we should be able to resend it back to us if we store the POST URL as well.

**Comment by annando** [comment]
We could do that, but surely this is a different issue ;-) 

And: Once a message was imported it couldn't be imported again.

**Comment by Hypolite** [comment]
Indeed, but why can't we import it again?

**Comment by annando** [comment]
The URI is already imported, so the storage function will reject it.

**Comment by Hypolite** [comment]
Gotcha, so the reimporter would need to clear the first item uri before importing again, I guess. It would be very useful to test federation over and over, or to fix broken items after a new fix.

**Pull request merged**